test/Streamly/Test/Data/Fold.hs

```diff
@@ -519,6 +519,27 @@ unzip =
     let v2 = (6, ["aa", "bb", "cc"])
     assert (v1 == v2)
 
+
+postscan :: Property
+postscan =
+    monadicIO $ do
+    v1 :: [Int] <-
+        run
+            $ S.fold (F.postscan FL.sum FL.toList)
+            $ S.fromList [1,2,3,4]
+    let v2 = [1,3,6,10]
```

Comment:
Can we generate a random list and compare against the list Data.List scan function, dropping the first element?